rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -1466,6 +1466,113 @@ def process_contamination_category_locally(self):
         return 0
 
 
+class IngestionClass(GenomicManifestBase):
+    """
+    Perform a targeted ingestion of AW1 or AW2 data
+    for an arbitrary set of participants based on genomic_set_member.id
+    """
+    def run(self):
+
+        # Validate arguments
+        if not self.args.csv and not self.args.member_ids:
+            _logger.error('Either --csv or --samples must be provided.')
```

Comment:
Should this say `--member-ids` instead of `--samples`?

Reply:
Yes, that was a late name change that got missed. Thanks!